src/App.tsx

```diff
@@ -1,7 +1,31 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends Component {
+  state = {
+    pressedKey: '',
+  };
+
+  componentWillMount() {
```

Comment:
because these are very different things. `componentWillMount` and `componentWillUnmount`
https://www.geeksforgeeks.org/what-is-componentwillmount-method-in-reactjs/
https://reactjs.org/docs/react-component.html#componentwillunmount